testing/hplip/APKBUILD

```diff
@@ -4,13 +4,13 @@
 # Contributor: Valery Kartel <valery.kartel@gmail.com>
 pkgname=hplip
 pkgver=3.18.6
-pkgrel=1
+pkgrel=2
 pkgdesc="Drivers for HP printers and scanners"
-arch="x86 x86_64" # missing sane on several arches
+arch="all"
 url="http://hplipopensource.com"
 license="GPL"
 depends=""
-makedepends="libjpeg-turbo-dev net-snmp-dev cups-dev libusb-dev sane-dev"
+makedepends="libjpeg-turbo-dev net-snmp-dev cups-dev libusb-dev sane-dev python-dev"
 subpackages="$pkgname-doc $pkgname-libs sane-backend-hpaio:sane"
```

Comment:
does it makes sense to create separate `py-sane` kinda package & how to deal with py3?

Reply:
Back when I tested this, the code worked with python3, i.e., instead of `python-dev`, this could also depend on `python3-dev`. However, at least the `hpps` executable has python (without version) in its shebang and thus breaks when you use python3. Iirc, changing the shebang fixed the issue and I could print with python 3. I found it cleaner to not to change the shebang but instead use python 2 for packaging. If you prefer python 3 I can also try to come up with a patch or sed expression that fixes the files for python 3. Note also that this does not affect sane. This is hplip only, i.e., if we split anything, this should be `py-hplip` (or similar). I have not tested scanning with hplip as I do not have a HP scanner but I assume this python library is only needed for printing. Note that for many printer drivers, `hpps` and thus this python library is not optional but they just do not work without it. See also this [related hplip bug](https://bugs.launchpad.net/hplip/+bug/1453104).